concepticondata/conceptlists/Gerardi-2021-244.tsv

```diff
@@ -0,0 +1,245 @@
+ID	NUMBER	ENGLISH	CONCEPTICON_ID	CONCEPTICON_GLOSS
+Gerardi-2021-244-1	1	ALL	98	ALL
+Gerardi-2021-244-2	2	ARM	1673	ARM
+Gerardi-2021-244-3	3	ARROW	977	ARROW
+Gerardi-2021-244-4	4	ASH	646	ASH
+Gerardi-2021-244-5	5	AXE	677	AXE
+Gerardi-2021-244-6	6	BACK	1291	BACK
+Gerardi-2021-244-7	7	BAD/EVIL	3764	BAD OR EVIL
+Gerardi-2021-244-8	8	BAMBOO	1927	BAMBOO
+Gerardi-2021-244-9	9	BANANA	868	BANANA
+Gerardi-2021-244-10	10	BASKET	1539	BASKET
+Gerardi-2021-244-11	11	BAT	1793	BAT
+Gerardi-2021-244-12	12	BE_WRONG	1390	WRONG
+Gerardi-2021-244-13	13	BEANS	2574	BEAN OR PEA
```

Comment:
823 BEAN

Reply:
Sorry: or do they have BEAN OR PEA in the other database?

Reply:
No, I see now: it is in fact BEAN, I think BEAN or PEA makes less sense (also for the other list).